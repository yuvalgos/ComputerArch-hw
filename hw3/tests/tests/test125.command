./dflow_calc tests/test125.dat tests/test125.in p64 p72 p31 p141 d17 p104 d179 p51 d150 d36 p141 p13 d109 p19 p145 p187 p183 d175 p127 p57 d38 d113 d130 d14 p159 p183 p145 d32 p159 p73 d152 p97 p193 d81 p105 d14 p130 p104 d100 d111 d41 d143 d148 p62 d12 p175 d34 d123 d104 d189 p2 p164 d19 d81 p199 d57 p87 p160 d108 d64 p170 p128 d31 p198 d37 p165 p86 p162 d154 p117 d190 d18