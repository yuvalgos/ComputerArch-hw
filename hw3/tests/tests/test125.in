2 25 15 14
6 4 20 3
1 9 22 18
1 16 0 2
2 4 15 31
1 2 6 25
3 22 0 18
7 2 19 25
0 0 1 19
5 30 29 16
5 7 4 3
6 23 24 5
0 1 18 1
1 31 16 29
6 3 12 6
2 11 12 7
1 15 25 12
5 0 22 13
5 17 3 31
2 11 31 3
6 15 26 0
7 26 3 13
3 9 12 4
2 11 31 9
6 24 0 29
2 15 24 0
6 6 5 27
1 5 20 11
6 22 10 28
5 16 31 25
6 17 20 23
3 0 22 30
5 9 26 6
6 10 29 3
1 18 18 10
4 8 17 14
6 7 14 17
2 30 0 1
6 7 27 18
4 24 3 12
1 17 6 23
7 1 12 3
0 4 10 17
2 30 6 2
5 6 12 8
2 0 30 31
8 8 12 15
0 8 22 4
0 28 29 31
0 12 31 23
5 29 8 30
5 22 30 6
0 1 22 18
2 18 25 7
6 24 24 7
8 15 17 19
3 16 30 17
2 10 0 14
6 12 26 11
1 23 5 19
0 13 13 8
1 31 13 16
0 21 31 10
4 12 9 30
6 22 3 12
7 20 2 4
3 15 11 5
4 11 2 24
5 17 26 8
6 15 8 29
5 31 11 19
7 5 9 8
5 22 30 14
7 26 1 6
4 4 11 13
3 31 1 19
5 23 9 4
6 11 26 17
3 5 25 5
2 2 3 12
2 6 5 23
7 13 10 10
0 11 13 19
5 19 29 1
0 9 21 11
3 23 21 27
8 13 5 22
8 1 28 28
7 18 16 6
3 24 25 7
8 5 5 7
4 25 27 18
8 14 28 23
8 1 4 16
2 27 7 18
3 3 1 7
7 21 26 20
1 13 16 20
3 29 6 17
7 29 0 11
5 18 17 8
8 17 26 14
8 31 29 0
2 25 23 8
8 12 6 9
1 14 18 27
4 5 17 11
3 30 2 12
3 21 14 4
8 21 12 18
4 6 25 5
8 27 21 12
8 13 16 22
1 7 2 27
0 25 19 10
0 7 5 19
5 21 6 22
4 11 6 14
1 22 12 16
0 26 22 24
8 15 5 18
1 4 24 18
3 4 17 8
8 27 5 17
3 24 28 3
3 27 10 28
1 30 11 29
8 3 31 23
2 16 16 7
5 20 10 15
7 27 22 8
6 16 26 0
7 25 29 11
1 24 4 22
6 22 22 13
7 2 8 26
2 13 5 14
7 23 12 22
1 12 9 17
2 29 0 3
2 30 29 18
6 13 7 13
4 14 5 4
2 22 26 31
5 24 9 17
0 6 10 4
1 21 17 23
3 1 17 27
3 23 3 5
2 1 22 15
4 3 1 25
1 27 9 18
1 4 10 27
4 23 3 30
1 21 21 5
3 2 19 9
8 20 21 9
5 28 19 25
3 4 6 13
0 20 3 14
6 0 2 11
6 29 5 25
6 16 15 7
6 9 4 20
4 24 20 4
8 30 14 7
3 19 19 20
0 16 7 31
5 23 17 18
5 30 14 26
0 20 11 6
8 24 16 30
8 9 4 13
0 18 14 30
6 21 11 5
5 10 19 0
6 11 2 10
1 4 22 25
0 11 0 29
1 15 18 19
1 9 31 27
0 4 14 17
5 23 15 13
2 12 27 4
1 4 10 27
3 14 15 22
4 19 4 28
1 1 13 29
2 27 12 21
4 3 2 6
6 8 9 7
8 0 19 12
2 15 2 16
8 12 9 22
4 8 26 6
5 28 3 14
0 4 8 18
7 14 12 8
7 26 17 0
8 19 10 29
