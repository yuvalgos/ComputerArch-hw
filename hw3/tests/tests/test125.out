Reading the opcodes latency info from tests/test125.dat ... Got latency for 9 opcodes
Reading the program file tests/test125.in ... Found 200 instructions
getProgDepth()==257
getDepDepth(64)==60
getDepDepth(72)==29
getDepDepth(31)==17
getDepDepth(141)==143
getInstDeps(17)=={6,-1}
getDepDepth(104)==111
getInstDeps(179)=={173,166}
getDepDepth(51)==58
getInstDeps(150)=={149,132}
getInstDeps(36)=={-1,30}
getDepDepth(141)==143
getDepDepth(13)==3
getInstDeps(109)=={104,100}
getDepDepth(19)==17
getDepDepth(145)==122
getDepDepth(187)==185
getDepDepth(183)==139
getInstDeps(175)=={166,160}
getDepDepth(127)==102
getDepDepth(57)==34
getInstDeps(38)=={-1,34}
getInstDeps(113)=={79,111}
getInstDeps(130)=={118,47}
getInstDeps(14)=={-1,-1}
getDepDepth(159)==179
getDepDepth(183)==139
getDepDepth(145)==122
getInstDeps(32)=={21,26}
getDepDepth(159)==179
getDepDepth(73)==73
getInstDeps(152)=={57,151}
getDepDepth(97)==55
getDepDepth(193)==144
getInstDeps(81)=={57,57}
getDepDepth(105)==135
getInstDeps(14)=={-1,-1}
getDepDepth(130)==137
getDepDepth(104)==111
getInstDeps(100)=={68,47}
getInstDeps(111)=={109,104}
getInstDeps(41)=={-1,14}
getInstDeps(143)=={119,102}
getInstDeps(148)=={127,106}
getDepDepth(62)==43
getInstDeps(12)=={-1,-1}
getDepDepth(175)==188
getInstDeps(34)=={-1,33}
getInstDeps(123)=={106,101}
getInstDeps(104)=={80,84}
getInstDeps(189)=={155,145}
getDepDepth(2)==0
getDepDepth(164)==181
getInstDeps(19)=={13,14}
getInstDeps(81)=={57,57}
getDepDepth(199)==201
getInstDeps(57)=={45,-1}
getDepDepth(87)==8
getDepDepth(160)==149
getInstDeps(108)=={105,74}
getInstDeps(64)=={14,63}
getDepDepth(170)==149
getDepDepth(128)==128
getInstDeps(31)=={28,9}
getDepDepth(198)==238
getInstDeps(37)=={31,12}
getDepDepth(165)==137
getDepDepth(86)==75
getDepDepth(162)==149
getInstDeps(154)=={146,106}
getDepDepth(117)==138
getInstDeps(190)=={180,115}
getInstDeps(18)=={14,13}
